src/services/bria/index.ts

```diff
@@ -214,8 +233,16 @@ export const NewOnChainService = (): INewOnChainService => {
 
       return response.getId() as PayoutId
     } catch (err) {
-      const errMsg = typeof err === "string" ? err : err.message
-      if (err.code == status.PERMISSION_DENIED && errMsg.includes("DestinationBlocked")) {
+      let errMsg = "Unknown error"
+      if (typeof err === "string") errMsg = err
```

Comment:
why do bria send a `string` error? is that a typical grpc thing?

Reply:
not sure. any ideas @dolcalmi? This is the simplest way i could figure out how to do this. I could not find a way to get a grpc error type.